src/components/App/App.js

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from 'react'
+import { useHistory } from 'react-router-dom'
+import styled from 'styled-components/macro'
+import { Route, Switch } from 'react-router-dom'
+import WorkoutPage from '../WorkoutPage'
+import MusicPage from '../MusicPage'
+import PlaylistPage from '../PlaylistPage'
+import data from '../tracksShort.json'
+import { rangeMin, rangeMax } from '../../services/cadenceRange'
+import convertCadenceTempo from '../../services/convertCadenceTempo'
+import roundedTempo from '../../services/roundedTempo'
+import { results } from '../staticWorkoutList.json'
+
+export default function App() {
+  const { push } = useHistory()
+
+  const [warmupSongs, setWarmupSongs] = useState([])
+  const [intervalsTSongs, setIntervalsTSongs] = useState([])
+  const [cooldownSongs, setCooldownSongs] = useState([])
+  const [playlist, setPlaylist] = useState([])
+
+  return (
+    <AppLayout>
+      <Switch>
+        <Route exact path="/">
+          <WorkoutPage results={results} />
+        </Route>
+        <Route
+          path="/music"
+          render={props => (
+            <MusicPage {...props} onCreatePlaylist={createPlaylist} />
+          )}
+        />
```

Comment:
It's totally valid to do this, however, it is a bit odd that this route is set different than the others. I personally would go with one way of writing it and stick to it. But that's just my opinion

Reply:
I agree with you. I´ll make it consistent.